MonoGame.Framework/GameWindow.cs

```diff
@@ -101,7 +101,9 @@ protected GameWindow()
 		/// <remarks>
 		/// This event is only supported on the Windows DirectX, Windows OpenGL and Linux platforms.
 		/// </remarks>
-		public event EventHandler<TextInputEventArgs> TextInput;
+		public event EventHandler<TextInputEventArgs> TextInput;
+        public event EventHandler<KeyInputEventArgs> KeyDown;
+        public event EventHandler<KeyInputEventArgs> KeyUp;
```

Comment:
@dellis1972 - Could we write up some docs for these methods.

In particular we should make clear when `TextInput` should be used and when `KeyUp/Down` should be used.